Formula/m/mdfried.rb

```diff
@@ -31,7 +31,7 @@ def install
       # Hello World
     MARKDOWN
 
-    output = shell_output("#{bin}/mdfried #{testpath}/test.md")
-    assert_match "system fonts detected", output
+    output = shell_output("#{bin}/mdfried #{testpath}/test.md 2>&1")
```

Comment:
💭 Should we do [the same as in `jnv`](https://github.com/Homebrew/homebrew-core/blob/490326fd7d25c91912a8ced4eecbb331ca387add/Formula/j/jnv.rb#L30-L34), that is, widen this test to not skip on Linux?

Reply:
no response from upstream yet, let's ship for now.

Reply:
@chenrui333 thank you for pinging me, here is fix in the works: #259086